### ISSUE TITLE
Can't install extensions automatically

### ISSUE THREAD

**Issue by IronLu233** [zeit/hyper#1951]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: MacOS sierria 10.12.5
- **Hyper.app version**:  1.3.3 (1.3.3.1754)

## Issue
Hyper is a awesome terminal app. When I use it. I found a bug when installing a extension.

First, I remove `~/.hyper_plugins/node_modules/` and clear all extension in `.hyper.js`

Then I start Hyper. In correct it works.

Next I add my extensions to `.hyper.js` and save it.

It occurs three errors. I restart it (using Cmd + Q). The errors still remain.

The notification is here
![image](https://user-images.githubusercontent.com/20639676/27366497-1b250aa2-5679-11e7-9eab-6a25fa2f310a.png)

**Of course I can open `~/.hyper_plugins/` and run `yarn` for installing all package manually. Then hyper will work without any errors**

Is it a bug when installing extensions?

and the dev tools console is:
```
(re)loading renderer plugins
bundle.js:1 Error: Cannot find module '/Users/iron/.hyper_plugins/node_modules/hyperpower'
    at Module._resolveFilename (module.js:455:15)
    at Function.Module._resolveFilename (/Applications/Hyper.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:403:25)
    at require (internal/module.js:20:19)
    at le.plugins.concat.map.ce (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:8876)
    at Array.map (native)
    at se (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:8830)
    at Object.<anonymous> (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:10365)
    at __webpack_require__ (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:119)
    at Object.<anonymous> (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:47:368728)le.plugins.concat.map.ce @ bundle.js:1
bundle.js:1 [Notification] Plugin load error: "hyperpower" failed to load in the renderer process. Check Developer Tools for details.
bundle.js:1 Error: Cannot find module '/Users/iron/.hyper_plugins/node_modules/hyperterm-monokai'
    at Module._resolveFilename (module.js:455:15)
    at Function.Module._resolveFilename (/Applications/Hyper.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:403:25)
    at require (internal/module.js:20:19)
    at le.plugins.concat.map.ce (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:8876)
    at Array.map (native)
    at se (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:8830)
    at Object.<anonymous> (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:10365)
    at __webpack_require__ (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:119)
    at Object.<anonymous> (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:47:368728)le.plugins.concat.map.ce @ bundle.js:1
bundle.js:1 [Notification] Plugin load error: "hyperterm-monokai" failed to load in the renderer process. Check Developer Tools for details.
bundle.js:1 Error: Cannot find module '/Users/iron/.hyper_plugins/node_modules/hyperline'
    at Module._resolveFilename (module.js:455:15)
    at Function.Module._resolveFilename (/Applications/Hyper.app/Contents/Resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:403:25)
    at require (internal/module.js:20:19)
    at le.plugins.concat.map.ce (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:8876)
    at Array.map (native)
    at se (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:8830)
    at Object.<anonymous> (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:10365)
    at __webpack_require__ (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:1:119)
    at Object.<anonymous> (file:///Applications/Hyper.app/Contents/Resources/app/dist/bundle.js:47:368728)le.plugins.concat.map.ce @ bundle.js:1
bundle.js:1 [Notification] Plugin load error: "hyperline" failed to load in the renderer process. Check Developer Tools for details.
index.html:35 total init time 280.96000000000004
3bundle.js:47 Unknown OSC code: "1337"
bundle.js:47 Unknown OSC code: "1"
bundle.js:47 Unknown OSC code: "133"
2bundle.js:47 Unknown OSC code: "1337"
2bundle.js:47 Unknown OSC code: "133"
```

**Comment by joggienl** [comment]
If you have got NPM5 this is related to #1888 and js allready in master now #1896

**Comment by IronLu233** [comment]
Yeah, I change the npm version to 4 and all problem has been solved! Thank you very much.
I hope next release will come later!
Awesome hyper.is!

**Comment by IronLu233** [comment]
fixed in #1896
